cli/src/helpers.rs

```diff
@@ -444,6 +444,17 @@ impl fmt::Display for ShowSolidoOutput {
             "  Total stSOL appreciation: {}",
             self.solido.metrics.st_sol_appreciation_sol_total
         )?;
+        writeln!(
+            f,
+            "  Total amount withdrawn:   {}, valued at {} when it was withdrawn",
```

Comment:
Nit: Might be better to add units into message, i..e "  Total amount withdrawn:   {} stSOL, valued at {} SOL when it was withdrawn"

Reply:
The units are included, they are part of the `Lamports` and `StLamports` display impl: https://github.com/ChorusOne/solido/blob/a601d3a8e3aabd4056ac7b14d8534b197706712e/program/src/token.rs#L61-L64

Reply:
Perfect!